server/src/test/java/io/crate/planner/operators/JoinPlanBuilderTest.java

```diff
@@ -111,9 +110,9 @@ public void testImplicitToExplicit_CrossJoinPairAlreadyExists() {
 
         assertThat(newJoinPairs).hasSize(1);
         JoinPair joinPair = newJoinPairs.get(0);
-        assertThat(joinPair.condition()).isSQL("(doc.t1.x = doc.t2.y)");
-        assertThat(joinPair.joinType()).isEqualTo(JoinType.INNER);
-        assertThat(remainingQueries).isEmpty();
+        assertThat(joinPair.condition()).isNull();
+        assertThat(joinPair.joinType()).isEqualTo(JoinType.CROSS);
+        assertThat(remainingQueries).hasSize(1);
```

Comment:
Do these test cases even make sense anymore given that the JoinPlanBuilder is no longer applying the cross -> inner join logic?

Reply:
👍